.github/workflows/pull-request-checks.yaml

```diff
@@ -20,6 +20,11 @@ jobs:
           sudo apt-get install --no-install-recommends -yq gcc gdb g++ maven jq flex bison libxml2-utils ccache cmake z3
       - name: Confirm z3 solver is available and log the version installed
         run: z3 --version
+      - name: Download cvc-5 from the releases page and make sure it can be deployed
+        run: |
+          wget https://github.com/cvc5/cvc5/releases/download/cvc5-0.0.7/cvc5-Linux
+          chmod u+x cvc5-Linux
+          $PWD/cvc5-Linux --version
```

Comment:
:no_entry_sign: Please rename the downloaded binary to just `cvc5` and add it to the path. Otherwise the regression tests would need to reference to the full path and operating system to run the solver. I guess we could either modify the path or move the binary to a directory which is already in the path to achieve this.
:information_source: The `z3` installation checks the version number as separate step in order to confirm that the environment change is persisted between steps and to make it easier for readers of the log to check the version number. Otherwise people reading the log would have to wade through the installation to get the version number.